app/Libraries/LocaleMeta.php

```diff
@@ -99,8 +99,8 @@ class LocaleMeta
             'name' => 'Tagalog',
             'flag' => 'PH',
         ],
-        'zh' => [
-            'name' => '中文',
+        'zh-CN' => [
```

Comment:
osu!wiki currently uses `zh`. Please check with them whether the wiki pages should be renamed to `zh-CN` as well or you rename this (and the translation files) to just `zh` or keep both (add new entry).